src/commands/synthetics/__tests__/utils.test.ts

```diff
@@ -169,6 +169,22 @@ describe('utils', () => {
       expect(handledConfig.startUrl).toBe(expectedUrl)
       process.env = envVars
     })
+
+    test('startUrl with empty variable is replaced', () => {
+      const publicId = 'abc-def-ghi'
+      const fakeTest = {
+        config: {request: {url: 'http://exmaple.org/path'}},
+        public_id: publicId,
+      } as Test
+      const configOverride = {
+        startUrl: 'http://127.0.0.1/newPath{{PARAMS}}',
```

Comment:
Where is `PARAMS` defined?

Reply:
It is extracted from the startUrl of the test (so in that case it is empty - it used to not be interpolated, but this PR fixes it and this test makes sure of that)